openshift/templates/backup/backup-conf-configmap_DeploymentConfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "kind": "ConfigMap",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "backup-conf",
+    "creationTimestamp": null
+  },
+  "data": {
+    "backup.conf": "# ============================================================\n# Databases:\n# ------------------------------------------------------------\n# List the databases you want backed up here.\n# Databases will be backed up in the order they are listed.\n#\n# The entries must be in one of the following forms:\n# - \u003cHostname/\u003e/\u003cDatabaseName/\u003e\n# - \u003cHostname/\u003e:\u003cPort/\u003e/\u003cDatabaseName/\u003e\n# - \u003cDatabaseType\u003e=\u003cHostname/\u003e/\u003cDatabaseName/\u003e\n# - \u003cDatabaseType\u003e=\u003cHostname/\u003e:\u003cPort/\u003e/\u003cDatabaseName/\u003e\n# \u003cDatabaseType\u003e can be postgres, mongo or mssql\n# \u003cDatabaseType\u003e MUST be specified when you are sharing a\n# single backup.conf file between postgres, mongo and mssql\n# backup containers.  If you do not specify \u003cDatabaseType\u003e\n# the listed databases are assumed to be valid for the\n# backup container in which the configuration is mounted.\n#\n# Examples:\n# - postgres=postgresql/my_database\n# - postgres=postgresql:5432/my_database\n# - mongo=mongodb/my_database\n# - mongo=mongodb:27017/my_database\n# - mssql=mssql_server:1433/my_database\n# -----------------------------------------------------------\n# Cron Scheduling:\n# -----------------------------------------------------------\n# List your backup and verification schedule(s) here as well.\n# The schedule(s) must be listed as cron tabs that \n# execute the script in 'scheduled' mode:\n#   - ./backup.sh -s\n#\n# Examples (assuming system's TZ is set to PST):\n# - 0 1 * * * default ./backup.sh -s\n#   - Run a backup at 1am Pacific every day.\n#\n# - 0 4 * * * default ./backup.sh -s -v all\n#   - Verify the most recent backups for all datbases\n#     at 4am Pacific every day.\n# -----------------------------------------------------------\n# Full Example:\n# -----------------------------------------------------------\n# postgres=postgresql:5432/TheOrgBook_Database\n# mongo=mender-mongodb:27017/useradm\n# postgres=wallet-db/tob_issuer\n# mssql=pims-db-dev:1433/pims\n#\n# 0 1 * * * default ./backup.sh -s\n# 0 4 * * * default ./backup.sh -s -v all\n# ============================================================\n0 1 * * * default ./backup.sh -s\nmssql=pims-db-dev:1433/pims"
```

Comment:
Is the formatting necessary?  It's fairly ugly to edit this way.

Reply:
If you use JSON you are forced to use `\n` for line breaks. YAML provides a nicer, human-readable alternative but I don't know if the backup tool supports that

Reply:
This file is 99% copied from the backup-conf file. Just the very bottom cron/connection string information is something I've added.